apps/settings/src/components/UserList/UserRowSimple.vue

```diff
@@ -147,8 +146,8 @@ export default {
 			}
 			return t('settings', '{size} used', { size: OC.Util.humanFileSize(0) })
 		},
-		canEdit() {
-			return getCurrentUser().uid !== this.user.id && this.user.id !== 'admin'
+		canEditUser() {
+			return (user) => this.settings.isAdmin || user.id !== OC.getCurrentUser().uid
```

Comment:
```suggestion
		canEditUser(user) {
			return this.settings.isAdmin || user.id !== getCurrentUser().uid
```

And move it to `methods` instead of having it as computed?

Reply:
But then again, I don't see why it needs to be a function in first place.
```suggestion
		canEditUser() {
			return this.settings.isAdmin || user.id !== getCurrentUser().uid
```

should be good enough

Reply:
Done.